<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>61</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>71</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditusername">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>311</width>
        <height>26</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Speicher meine Sitzung</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Login</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Update/Fetch</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lblStatus">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Nicht eingeloggt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPassword">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>311</width>
        <height>26</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Update-Intervall (min)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxUpdateInterval">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>143</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Benachrichtigung (sec)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxNotificationDuration">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>143</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>145</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Chatbox Nachrichten?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxChatBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>166</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>137</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Private Nachrichten?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxPM">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>229</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>74</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Darkmode?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxDarkMode">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>303</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Cookies verschleiern?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnResetSettings">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>101</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Minimiert starten?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxMinimize">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ticker f√ºr: &lt;/span&gt;&lt;a href=&quot;https://free-hack.com&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#007af4;&quot;&gt;free-hack.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Github: &lt;/span&gt;&lt;a href=&quot;https://github.com/DailoxFH/Free-Hack-Ticker-Linux&quot;&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline; color:#007af4;&quot;&gt;github.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Made by Dailox&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditusername</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spinBoxUpdateInterval</tabstop>
  <tabstop>spinBoxNotificationDuration</tabstop>
  <tabstop>checkBoxChatBox</tabstop>
  <tabstop>checkBoxPM</tabstop>
  <tabstop>checkBoxDarkMode</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>btnResetSettings</tabstop>
  <tabstop>checkBoxMinimize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
